p2p/client.go

```diff
@@ -171,12 +174,6 @@ func (c *Catchup) sendSyncRequest(peer *Peer) error {
 		zap.Uint32("startBlock", c.requestedStartBlock),
 		zap.Uint32("endBlock", c.requestedEndBlock))
 
-	err := peer.SendSyncRequest(c.requestedStartBlock, c.requestedEndBlock+1)
-
-	if err != nil {
-		return fmt.Errorf("send sync request: %s", err)
-	}
-
-	return nil
-
+	return errors.Wrapf(peer.SendSyncRequest(c.requestedStartBlock, c.requestedEndBlock+1),
```

Comment:
Even if this is equivalent to previous code, the previous version is explicit and helps document the code that we return an error only when `SendSyncRequest` returns one. A new developer needs to understand the semantic that `Wrapf` returns `nil` when the received error is `nil` which might not be obvious at first sight.

Please revert to old style of course keeping `errors.Wrapf` call.